revivelib/core.go

```diff
@@ -169,7 +169,8 @@ func (r *Revive) Format(
 	return output, exitCode, nil
 }
 
-func getPackages(includePatterns []string, excludePatterns ArrayFlags) ([][]string, error) {
+// GetPackages yields the packages and corresponding files for the given patterns
+func GetPackages(includePatterns []string, excludePatterns ArrayFlags) ([][]string, error) {
```

Comment:
Could me move this to internal, so it's not used outside the module?

Reply:
yes, I'm undoing the "let's use patterns in tests" path